typings/index.d.ts

```diff
@@ -1755,6 +1765,43 @@ declare module 'discord.js' {
     public readonly createdAt: Date;
   }
 
+  export class Sticker extends Base {
+    constructor(client: Client, data: unknown);
+    public readonly createdTimestamp: number;
+    public readonly createdAt: Date;
+    public available: ?boolean;
```

Comment:
```suggestion
    public available: boolean | null;
```

Reply:
already discussed with Vlad, both work

Reply:
@advaith1 It's better for consistency since `| null` is used everywhere else in the typings. Also using the `?` prefix will cause it to fail to compile:

![image](https://user-images.githubusercontent.com/18150845/124066490-082ba080-da39-11eb-95d7-8379a470178c.png)